src/System.Private.CoreLib/src/System/Array.cs

```diff
@@ -2157,6 +2186,27 @@ public static int FindLastIndex<T>(T[] array, int startIndex, int count, Predica
 
         public int GetLowerBound(int dimension)
         {
+#if REAL_MULTIDIM_ARRAYS
+            if (!this.EETypePtr.IsSzArray)
+            {
+                if ((dimension >= Rank) || (dimension < 0))
+                    throw new IndexOutOfRangeException();
+
+                unsafe
+                {
+                    fixed (int* pNumComponents = &_numComponents)
+                    {
+                        // Lower bounds follow after upper bounds.
+#if BIT64
+                        // _numComponents is padded on 64-bit systems.
+                        return *(pNumComponents + 1 + 1 + Rank + dimension);
```

Comment:
I think it would look better to make it a constant - since we have ifdefed constants in this file anyway:

`return *(pNumComponents + DIMENSIONS_DELTA + Rank + dimension);`